homeassistant/components/tolo/const.py

```diff
@@ -5,13 +5,3 @@
 
 DEFAULT_RETRY_TIMEOUT = 1
 DEFAULT_RETRY_COUNT = 3
```

Comment:
Are these still used?

Reply:
Yes, it's imported by `__init__.py` and used for initalizing `ToloClient`.